unixfs/io/dagreader.go

```diff
@@ -65,6 +65,8 @@ func NewDagReader(ctx context.Context, n *mdag.Node, serv mdag.DAGService) (Read
 	case ftpb.Data_Directory:
 		// Dont allow reading directories
 		return nil, ErrIsDir
+	case ftpb.Data_Raw:
+		fallthrough
```

Comment:
i mean, technically this works... id still like to see it special cased, to avoid making a child context and calling GetDAG.